enterprise/frontend/src/embedding-sdk/lib/theme/get-embedding-theme.unit.spec.ts

```diff
@@ -18,6 +18,11 @@ describe("Transform Embedding Theme Override", () => {
           "text-tertiary": "green",
           "background-disabled": "pink",
         },
+        components: {
+          popover: {
+            zIndex: undefined,
+          },
+        },
```

Comment:
Modified this test to ensure that even if we provide `zIndex: undefined`, we would still get an appropriate `zIndex: 200` default value back. Expand the diff to see how that works.